### DISCUSSION TITLE
administrator account setting

### DISCUSSION THREAD

**Discussion by kivakin** [gocd/gocd#12999] [Q&A]
Hi, i have qusetion, i need your help.
I'm done setting up the pipeline gocd-server and agent v24.2.0.

Now I want to create an administrator account to prevent unauthenticated users from modifying the pipeline by accessing http://127.0.0.1:8153/go/ from outside. I'm not sure what to do for this.

? Question 1
I'm trying to add and account for authenticated. is that right? 

? Question 2
how can i import User for login or authenticated ? should i use LDAP plugin?

`ADMIN > Users Management > Import User`

**Comment by chadlwilson** [comment]
Please read https://docs.gocd.org/current/configuration/dev_authentication.html - there many options that depend on what you have available in your environment.
